package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "argo-clinical",
-  "version": "1.82.0",
+  "version": "1.82.2",
```

Comment:
should this be one?

Reply:
We did an RC 1.82.1 last week which for some reason did not include the version update
https://github.com/icgc-argo/argo-clinical/pull/1033
The 1.82.1 branch exists so I just continued to 1.82.2